client/src/reducers/index.js

```diff
@@ -4,6 +4,7 @@ import thunk from "redux-thunk";
 import cascadeReducers from "./cascade";
 import undoable from "./undoable";
 import config from "./config";
+import userinfo from "./userinfo";
```

Comment:
This is breaking the [airbnb style guide](https://airbnb.io/javascript/#naming--filename-matches-export).(lowercase import/filename, PascalCase export)

Reply:
Hmm... I was matching the pattern that existed in reducers but also noticed it looked odd. Open an issue to change them all?

Reply:
Will do 👍